src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -60,49 +70,45 @@ void WriteObjectsToMemoryStore(
 CoreWorkerDirectActorTaskSubmitter::CoreWorkerDirectActorTaskSubmitter(
     rpc::ClientCallManager &client_call_manager,
     std::shared_ptr<CoreWorkerMemoryStoreProvider> store_provider)
-    : client_call_manager_(client_call_manager), in_memory_store_(store_provider) {}
+    : client_call_manager_(client_call_manager),
+      in_memory_store_(store_provider),
+      resolver_(in_memory_store_) {}
```

Comment:
I think it's probably time we added a unit test for DirectActorTaskSubmitter. It doesn't look too bad, you just need a ClientFactory mock similar to the task submitter.